modular_skyrat/modules/cortical_borer/code/cortical_borer_abilities.dm

```diff
@@ -421,6 +417,9 @@
 			continue
 		potential_freezers += listed_human
 	if(length(potential_freezers) == 1)
+		. = ..()
+		if(. == FALSE)
+			return FALSE
```

Comment:
Okay I'm tired of writing the same suggestion over and over, I think you get it by now.
```suggestion

		. = ..()
		if(!.)
			return FALSE

```